bitnami/kong/templates/deployment.yaml

```
@@ -0,0 +1,268 @@
+      app.kubernetes.io/component: server
+  {{- if .Values.updateStrategy }}
+  strategy: {{- toYaml .Values.updateStrategy | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels: {{- include "kong.labels" . | nindent 8 }}
+        app.kubernetes.io/component: server
+      annotations:
+        {{- if (include "kong.createExternalDBSecret" .) }}
+        checksum/secret: {{ include (print $.Template.BasePath "/external-database-secret.yaml") . | sha256sum }}
+        {{- end }}
+        checksum/configmap-kong: {{ include (print $.Template.BasePath "/kong-script-configmap.yaml") . | sha256sum }}
+        {{- if .Values.metrics.enabled }}
+        checksum/configmap-metrics-plugin: {{ include (print $.Template.BasePath "/metrics-script-configmap.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if .Values.podAnnotations }}
+        {{- include "kong.tplValue" (dict "value" .Values.podAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- if .Values.ingressController.enabled }}
+      serviceAccountName: {{ include "kong.serviceAccount" . }}
+      {{- end }}
+      {{- if .Values.podSecurityContext }}
+      securityContext: {{- include "kong.tplValue" (dict "value" .Values.podSecurityContext "context" $) | nindent 8 }}
+      {{- end }}
+      {{- include "kong.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName | quote }}
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{- include "kong.tplValue" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- include "kong.tplValue" (dict "value" .Values.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- include "kong.tplValue" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+    {{- if .Values.initContainers }}
+      initContainers:
+      {{- include "kong.tplValue" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+    {{- end }}
+      containers:
+        - name: kong
+          image: {{ template "kong.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.containerSecurityContext }}
+          securityContext: {{- include "kong.tplValue" (dict "value" .Values.containerSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.kong.command }}
+          command: {{- include "kong.tplValue" (dict "value" .Values.kong.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.kong.args }}
+          args: {{- include "kong.tplValue" (dict "value" .Values.kong.args "context" $) | nindent 12 }}
+          {{- end }}
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "kong quit"]
```

In the `kong-ingress-controller` container you're using a different format to define the command: 
```suggestion:
                command:
                  - bash
                  - -ec
                  - kong quit
````

---

Done